tests/test_requirements.txt

```diff
@@ -1,15 +1,15 @@
-astropy==3.0.5
+astropy==3.2.3
 codecov==2.0.15
-coverage==4.5.1
+coverage==4.5.4
 fastcache==1.0.2
 glueviz==0.13.3
 h5py==2.10.0
 ipython==7.1.1
-matplotlib==3.1.3
+matplotlib==3.3.0
 mock==2.0.0; python_version < '3.0'
-nose-timer==0.7.3
+nose-timer==1.0.0
 nose==1.3.7
-pandas==0.23.4
```

Comment:
I'm guessing you have a good reason for this downgrade but I couldn't find it in the commit messages, just curious :)

Reply:
0.23.4 -> 0.25.3 is not a downgrade ;)

Reply:
oops, I only saw the third digit and then my brain shut down, apparently 😅

Reply:
Although that reminds me that pandas 1.0 came out almost a year ago... I would naively expect the latest version to be more efficient, do you know if it is ?

Reply:
I think the idea behind `test_requirements.txt` is to have the earliest possible version that works. I don't entirely agree with that, but it is what it is. We should probably have a separate test build where all those things are completely unpinned.

Reply:
Yes, that would help catch breaking updates from our deps early. May be worth keeping this in mind next time we discuss the testing infrastructure as a whole :)
thank you for clarifying this !